lib/ui/home/home_empty_state.dart

```diff
@@ -43,7 +44,7 @@ class HomeEmptyStateWidget extends StatelessWidget {
                     SizedBox(
                       width: 400,
                       child: OutlinedButton(
-                        onPressed: onScanTap,
+                        onPressed: PlatformUtil.isMobile() ? onScanTap : null,
```

Comment:
Let's hide this option on desktop?

Reply:
Same for the speed dial as well. We could open the manual entry page directly when the user clicks on `+`